app/controllers/admin/api/cms/base_controller.rb

```diff
@@ -3,6 +3,10 @@ class Admin::Api::CMS::BaseController < Admin::Api::BaseController
   before_action :deny_on_premises_for_master
   self.access_token_scopes = %i[cms account_management]
 
+  class_action_on_unpermitted_parameters :raise
+  class_always_permitted_parameters %i[id page per_page]
```

Comment:
`class` makes me wonder if those configurations may be used anywhere. I think we could prepend "controller" for better understanding or even remove it altogether so it looks closed to the Rails config option.

```suggestion
  controller_action_on_unpermitted_parameters :raise
  controller_always_permitted_parameters %i[id page per_page]
```
or even

```suggestion
  action_on_unpermitted_parameters :raise
  always_permitted_parameters %i[id page per_page]
```

**Warning:** This comment smells a little too much of _personal preference_ so feel extra free to discard it.